src/app/line-chart/line-chart.component.ts

```diff
@@ -0,0 +1,63 @@
+import { Component } from '@angular/core';
+import * as Highcharts from 'highcharts';
+import ExportingModule from 'highcharts/modules/exporting';
+import HC_customEvents from 'highcharts-custom-events';
+ExportingModule(Highcharts);
+HC_customEvents(Highcharts);
+
+@Component({
+  selector: 'app-line-chart',
+  templateUrl: './line-chart.component.html',
+  styleUrls: ['./line-chart.component.css']
+})
+export class LineChartComponent {
+
+  // For all demos:
```

Comment:
This comment is no longer true. Even if the same instance across components will be used the Highcharts local property is set for each <type>ChartComponent - unlike before where this was a single component and the same property was reused.